src/Storages/MergeTree/DataPartStorageOnDisk.cpp

```diff
@@ -861,12 +861,13 @@ std::string DataPartStorageBuilderOnDisk::getRelativePath() const
 
 void DataPartStorageBuilderOnDisk::createDirectories()
 {
-    return volume->getDisk()->createDirectories(fs::path(root_path) / part_dir);
+    LOG_INFO(&Poco::Logger::get("DEBUG"), "CREATING DIRECTORY {}", (fs::path(root_path) / part_dir).string());
```

Comment:
Is this the intention or you just forgot to remove?

Reply:
I have the same question :)